router/batchrouter/handle_async.go

```diff
@@ -73,9 +73,8 @@ func getPollInput(job *jobsdb.JobT) common.AsyncPoll {
 	return common.AsyncPoll{ImportId: importId}
 }
 
-func enhanceErrorResponseWithFirstAttemptedAtt(msg stdjson.RawMessage, errorResp []byte) (time.Time, []byte) {
-	firstAttemptedAt := getFirstAttemptAtFromErrorResponse(msg)
-	return firstAttemptedAt, routerutils.EnhanceJsonWithTime(firstAttemptedAt, "firstAttemptedAt", errorResp)
+func enhanceErrorResponseWithFirstAttemptedAtt(msg stdjson.RawMessage, errorResp []byte) []byte {
```

Comment:
The fact that the response we're enhancing is an error one doesn't seem to be relevant.

```suggestion
func enhanceResponseWithFirstAttemptedAt(msg stdjson.RawMessage, resp []byte) []byte {
```

Reply:
True
I think the function was named as such because it's only used to enhance `jobsdb.JobStatus`'s `ErrorResponse` field.
Changed it anyway.